README.md

```diff
@@ -65,6 +65,17 @@ To fetch dependencies and get cooking:
 | `test` | Runs unit and functional tests.
 | `validate` | Alias for `grunt validate-shrinkwrap` task (ignoring any errors which may be reported).
 
+### Creating Dummy Data
+
+If you just want to test something quickly with a small, known test data set:
+
+1. run `db/create_db.js` to drop and re-create the local mysql db
```

Comment:
Should this actually be `server/db/create_db.js` or indicate that each of these are executed within the server dir?